app/components/m-header.hbs

```diff
@@ -1,87 +1,168 @@
+      <WebComponents::AuButtonLink
+              @skin="muted"
+              @icon="ki ki-circle-question"
+              @layout="icon-left">
+        {{t "help"}}
+      </WebComponents::AuButtonLink>
+      <AttachPopover
+              @renderInPlace={{true}}
+              @class="ember-attacher-popper"
+              @hideOn="clickout click"
+              @showOn="click"
+              @animation="shift"
+              @placement="bottom" {{!-- as |attacher| --}}
+      >
+        <WebComponents::AuDropdown>
+          <WebComponents::AuDropdown::Item @href="https://kaleidos.vlaanderen.be/Kaleidos_Raadplegen_20200629.pdf" target="_blank">
+            {{t "look-at-documentation"}}
+          </WebComponents::AuDropdown::Item>
+          <WebComponents::AuDropdown::Item  @href="mailto:kaleidos.servicedesk@vlaanderen.be">
+            <div>
+              <span>{{t "contact-us"}}</span>
+              <span class="auk-u-muted auk-u-text-small">{{t "email-servicedesk"}}</span>
             </div>
-            <a href="" {{action "logout"}}>
+          </WebComponents::AuDropdown::Item>
+        </WebComponents::AuDropdown>
+      </AttachPopover>
+    </div>
+    <div class="auk-toolbar-complex__item auk-u-mr-3">
+      {{#if currentSession.userContent}}
+        <WebComponents::AuButtonLink
+                data-test-m-header-user-actions
+                @skin="muted"
+                @icon="chevron-down"
+                @layout="icon-right">
+          {{currentSession.userContent.firstName}}
+        </WebComponents::AuButtonLink>
+        <AttachPopover
+                @renderInPlace={{true}}
+                @class="ember-attacher-popper"
+                @hideOn="clickout click"
+                @showOn="click"
+                @animation="shift"
+                @placement="bottom" {{!-- as |attacher| --}}
+        >
+          <WebComponents::AuDropdown>
+            <WebComponents::AuDropdown::Item @textOnly={{true}}>
+              <div class="auk-avatar-and-text">
+                <div class="auk-avatar auk-avatar--default auk-avatar--regular">
+                  <div class="auk-icon auk-icon--default auk-icon--regular">
+                    <i class="ki ki-user"></i>
+                  </div>
+                </div>
+                <div class="auk-avatar-and-text__text">
+                  <p>{{currentSession.userContent.firstName}} {{currentSession.userContent.lastName}}</p>
+                  <p class="auk-u-muted auk-u-text-small">{{currentSession.userRole}}</p>
+                </div>
+              </div>
+            </WebComponents::AuDropdown::Item>
+            <WebComponents::AuDropdown::Divider />
+            <WebComponents::AuDropdown::Item >
+              {{!-- TODO link naar mijn profiel => nog niet bestaande--}}
+              {{t "my-profile"}}
+            </WebComponents::AuDropdown::Item>
+            <WebComponents::AuDropdown::Item data-test-m-header-user-action-logout {{action "logout"}}>
```

Comment:
De `action` helper is oude Ember syntax die vervangen moet worden. Je zou hier een click-handler kunnen gebruiken `{{on "click" this.logout}}`, maar ik denk dat het beter is om de `AuDropdown::Item` component uit te breiden.

Op dit moment aanvaardt die als argumenten `@href` of `@route`, waarna een anchor tag gerenderd wordt ofwel met `<a>` ofwel via `<LinkTo>` die linkt naar een andere route/pagina. Volgens mij is de 3e optie die we hier nodig hebben een item met een `@onClick` action handler dat zich gedraagt als een knop. Die kan dan gebruikt worden voor deze logout action.